account_promissory_note_bankia/report/check_print.py

```diff
@@ -0,0 +1,33 @@
+from . import lang_ES
+from odoo import api, models
+
+
+class ReportPromissoryNotePrint(models.AbstractModel):
```

Comment:
This abstract class is better put in account_payment_promissory_note, is it posible??